### PR TITLE
Set value/ Check for DirtyCHild  Implementation

### PR DESCRIPTION
**Pull request by bibekluitel**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -35,45 +32,82 @@ describe('VCOnstructor Test', function() {
 });
 
 describe('verifies the set and get operations', function() {
+
```
Comment (arshadkazmi42) [inline test/test.js]: Should we keep these line breaks after every line?
I like keeping all the lines in test without spaces.
there should be line break only for each `describe` and `it`
Whats your opinion on this?

File: index.js [hunk 2]
```diff
@@ -26,22 +29,25 @@ function JMON(data) {
...
+  this.isCreated = false;
+  this.isUpdated = false;
+  this.isDeleted = false;
+};
+
+/**
+ * Returns true if self or any of children has been modified.
+ * Returns false if nothing has been changed.
+ */
+JMON.prototype.checkIfUpdated = function() {
```
Comment (arshadkazmi42) [inline index.js]: I think we should name this function to `isModified`. That will be better. `checkIfUpdated` sounds a bit odd.
Any inputs on this?

File: test/test.js (hunk 1 above)
Comment (bibekluitel) [reply test/test.js]: Ya sure . i actually  dont have an idea regarding that. i just gave line so that i will be easy.

File: index.js (hunk 2 above)
Comment (bibekluitel) [reply index.js]: ya  that sounds better.

**Pull request merged**